### DISCUSSION TITLE
cant update geometry because the number of points is higher then the old pcd.

### DISCUSSION THREAD

**Discussion by MeyerBuaharon** [isl-org/Open3D#5229] [Q&A]
When I try to do updateGeometry() on my new pcd it gives me the following warning:
[Open3D WARNING] Geometry for point cloud points cannot be updated because the number of points exceeds the existing point count (Old: 97098, New: 1000000)

Does anyone know how I can bypass that?

**Comment by Gaussiandra** [comment]
Also have this problem(
